packages/mirrorful/editor/src/components/ColorPalette/AddColorSkeleton.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+    Box,
+    Button,
+    Stack,
+    Text,
+    useDisclosure,
+    Badge,
+    Heading,
+    Divider,
+    Menu,
+    MenuButton,
+    MenuList,
+    MenuItem,
+    MenuItemOption,
+    MenuGroup,
+    MenuOptionGroup,
+    MenuDivider,
+    IconButton,
+    Icon,
+  } from '@chakra-ui/react'
+  import tinycolor from 'tinycolor2'
+  import { generateDefaultColorShades, handleInvalidColor } from './utils'
+  import { TColorData, TColorVariant } from 'types'
+
+  // Accepts: backgroundColorString, a valid CSS color string, eg '#EEE' or 'steel-blue', which sets the color of row elements.  An invalid color results in random assignment
```

Comment:
While I understand the idea of a backgroundColorString, I feel like as it's a skeleton, it should just be grey and does not need to be flexible and we can remove that as a prop.

Reply:
Seems fair, the thinking was if the site styling changed in future, this would allow this to fit in, but agreed, it removes a lot of complications making it static...